### PR TITLE
Backport support for Devuan OS in Stable 2.7

### PR DESCRIPTION
**Pull request by vpiserchia**
##### SUMMARY
This branch incorporates the Devuan support from 4e9b8136c2b48b37ab103b763976cfbaebad5db4

##### ISSUE TYPE
- Backport Pull Request
- Feature Pull Request

##### COMPONENT NAME
This patch series involves the setup module

##### ADDITIONAL INFORMATION
This patch series backport support for Devuan based distro

### CODE REVIEW

File: lib/ansible/release.py [hunk 1]
```diff
@@ -19,6 +19,6 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-__version__ = '2.7.2.post0'
+__version__ = '2.7.2.1'
```
Comment (mattclay) [inline lib/ansible/release.py]: Release versions are changed during the release process and should not be changed in pull requests.

**Review by mattclay — CHANGES_REQUESTED** [review]

**Comment by abadger** [comment]
In addition to mattclay's note, this needs to have a changelog fragment added for this bugfix.  you can look in https://github.com/ansible/ansible/tree/devel/changelogs/fragments  for examples from other changelogs.

File: lib/ansible/release.py (hunk 1 above)
Comment (vpiserchia) [reply lib/ansible/release.py]: ok, I'll handle this

File: changelogs/.changes.yaml [hunk 2]
```diff
@@ -627,4 +627,5 @@ releases:
     - win_uri-junk-data.yaml
     - yum-handle-obsoletes-check-update.yaml
     - yumdnf-autoremove.yaml
+    - 49123-backport-devuan-support.yml
```
Comment (mattclay) [inline changelogs/.changes.yaml]: This file is updated automatically during a release and should not be manually updated.

**Review by mattclay — CHANGES_REQUESTED** [review]

File: changelogs/.changes.yaml (hunk 2 above)
Comment (vpiserchia) [reply changelogs/.changes.yaml]: so all I need is the file "49123-backport-devuan-support.yml" and nothing more?

Comment (Akasurde) [reply changelogs/.changes.yaml]: yes, remove reference from `changelogs/.changes.yaml` and leave `changelogs/fragments/49123-backport-devuan-support.yml` as is.

Comment (vpiserchia) [reply changelogs/.changes.yaml]: done, thank  you for your patience

**Comment by vpiserchia** [comment]
@Akasurde, I fixed the yamllint error, please review it, thanks

**Comment by Akasurde** [comment]
@vpiserchia Review is already done. 2.7 Release manager will merge this PR. Thanks.

**Comment by abadger** [comment]
Merged for the 2.7.5 release.

**Pull request merged**